packages/back-end/src/controllers/features.ts

```diff
@@ -535,9 +605,17 @@ export async function postFeaturePublish(
   req.checkPermissions("manageFeatures", feature.project);
 
   const revision = await getRevision(org.id, feature.id, parseInt(version));
+  const user = org.members.find((member) => member.id === userId);
+  const isAdmin = user?.role === "admin";
```

Comment:
Instead of checking the role directly, we should make a new permission called `bypassApprovalChecks`, grant that to the admin role, and then check the permission here.  We are going to add support for custom roles soon, so want to avoid referencing roles directly.

Also, users can be part of teams, so the roles get complicated.  The logic here is only checking their individual global role, which won't work.  Switching to permission checks will fix this too.